src/types/types.ts

```diff
@@ -1093,6 +1093,17 @@ export interface EdgeSwapQuote {
   readonly request: EdgeSwapRequest
 
   readonly isEstimate: boolean
+
+  /**
+   * This quote may be partially fulfilled with remaining source funds left
+   * in wallet
+   */
+  readonly isMaybePartial?: boolean
+
+  /**
+   * Maximum amount of time this quote will take to be fulfilled (in seconds)
+   */
+  readonly maxFulfillmentTime?: number
```

Comment:
I would like `maxFulfillmentSeconds`, so the name caries its own units.

Reply:
sounds good